src/Bundle/Tests/DependencyInjection/GridBuilderConfigurationTest.php

```diff
@@ -903,6 +904,37 @@ public function it_builds_extended_grids_with_grids_as_service(): void
         ]);
     }
 
+    /**
+     * @test
+     */
+    public function it_builds_grid_with_a_grid_attribute(): void
+    {
+        $grid = new AttributeFooGrid();
+
+        $this->load([
+            'grids' => [
+                'app_foo_with_attribute' => $grid->toArray(),
```

Comment:
Should test cover case where name is not specified and FQCN is used instead?

Reply:
Smart ! PR updated